ee/clickhouse/models/test/__snapshots__/test_property.ambr

```diff
@@ -138,3 +138,15 @@
     },
   )
 ---
+# name: test_parse_prop_clauses_precalculated_cohort
+  <class 'tuple'> (
+    '
+      AND ( pdi.person_id IN (
```

Comment:
this here is the key - this snapshot didn't exist but if you look on Sentry you'll see that this `pdi.` isn't here on the query to `cohortpeople`

https://sentry.io/organizations/posthog/issues/3842781226/?project=1899813&referrer=github_integration